_posts/2017-01-19-new-pulp-logo.md

```diff
@@ -0,0 +1,30 @@
+---
+title: New Pulp Logo
+author: David Davis
+---
+
+![New Pulp Logo](/images/pulp_logo_big.png)
```

Comment:
When I rendered this on the site it didn't show correctly for me. It just showed the unclickable text "New Pulp Logo".